README.md

```diff
@@ -97,6 +97,17 @@ Port number the server runs on.
 
 Defaults to `8080`.
 
+###### -S, --secure
+Specify that express should serve over HTTPS.
+
+Disabled by default.
+
+###### -C, --certificatePath
+SSL Certificate Chain path for express server.
```

Comment:
Fix capitalization and don't mention express for reasons stated earlier.

`SSL Certificate Chain path for express server.` -> `SSL certificate chain for the server.`